doc/build-windows.md

```diff
@@ -68,7 +72,7 @@ Then build using:
     cd ..
     ./autogen.sh # not required when building from tarball
     CONFIG_SITE=$PWD/depends/x86_64-w64-mingw32/share/config.site ./configure --prefix=/
-    make
+    make -j4
```

Comment:
nit: not sure if we should set the `jobs` amount in the documentation. IMO we also don't do that for other build docs. It depends on available RAM/cores.